pkg/tasks/prerequisites.go

```diff
@@ -50,6 +50,7 @@ func kubeadmPreflightChecks(s *state.State) error {
 			ctx.Logger.Info("	preflight...")
 			_, _, err := ctx.Runner.Run(heredoc.Docf(`
 				sudo kubeadm init phase preflight \
+					--ignore-preflight-errors=DirAvailable--var-lib-etcd,FileAvailable--etc-kubernetes-manifests-kube-apiserver.yaml,FileAvailable--etc-kubernetes-manifests-kube-controller-manager.yaml,FileAvailable--etc-kubernetes-manifests-kube-scheduler.yaml,FileAvailable--etc-kubernetes-manifests-etcd.yaml,Port-6443,Port-10259,Port-10257,Port-10250,Port-2379,Port-2380 \
```

Comment:
shouldn't we use kubeadm config? and we already partually have some

```golang
	nodeRegistration := newNodeRegistration(s, host)
	nodeRegistration.IgnorePreflightErrors = []string{
		"DirAvailable--var-lib-etcd",
		"DirAvailable--etc-kubernetes-manifests",
		"ImagePull",
	}
```

Reply:
No, that would apply ignore list to `kubeadm init` as well. We only want to apply this ignore list to `kubeadm init` running preflight checks.